client/src/app/containers/Home/ActiveGamesTable.jsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+import { useInjectReducer, useInjectSaga } from 'redux-injectors';
+import { selectCurrentUser } from '../../../auth/selectors';
+import { authSaga } from '../../../auth/saga';
+import {
+  sliceKey as authSliceKey,
+  reducer as authReducer,
+} from '../../../auth/slice';
+import { gameSaga } from '../Game/saga';
+import {
+  sliceKey as gameSliceKey,
+  reducer as gameReducer,
+  actions as gameActions,
+} from '../Game/slice';
+import { selectActiveGames } from './selectors';
+import { homeSaga } from './saga';
+import {
+  sliceKey as homeSliceKey,
+  reducer as homeReducer,
+  actions as homeActions,
+} from './slice';
+
+const ActiveGamesTable = () => {
+  useInjectReducer({ key: authSliceKey, reducer: authReducer });
+  useInjectSaga({ key: authSliceKey, saga: authSaga });
+  useInjectReducer({ key: homeSliceKey, reducer: homeReducer });
+  useInjectSaga({ key: homeSliceKey, saga: homeSaga });
+  useInjectReducer({ key: gameSliceKey, reducer: gameReducer });
+  useInjectSaga({ key: gameSliceKey, saga: gameSaga });
+  const history = useHistory();
+  const dispatch = useDispatch();
+  const currentUser = useSelector(selectCurrentUser);
+  const activeGames = useSelector(selectActiveGames(currentUser.id));
+
+  // event handlers
+  const onRefreshActiveGames = () => {
+    dispatch(homeActions.refreshActiveGames({ id: currentUser.id }));
+  };
+  const onGoToGame = gID => {
+    dispatch(gameActions.goToGame(gID, history));
+  };
+
+  let gameButtons = activeGames.map(activeGame => {
+    if (!activeGame || !activeGame.gameID) {
+      return;
+    }
+    const gID = activeGame.gameID;
+
+    let opponents = [];
```

Comment:
💡 Can we let the server do the heavy lifting and add an `Opponents` field to the response?